picard/ui/mainwindow/__init__.py

```diff
@@ -288,7 +288,7 @@ def keyPressEvent(self, event):
             else:
                 self.remove_selected_objects()
         elif event.matches(QtGui.QKeySequence.StandardKey.Find):
-            self.search_edit.setFocus(QtCore.Qt.FocusReason.ShortcutFocusReason)
+            self.toggle_find_boxes()
```

Comment:
I think using the search shortcut for the filters by default is useful, and especially with a version 3 release this change can be well reasoned for.

But maybe we should keep a shortcut for the search box as well. Maybe something like `Ctrl+Shift+F`. @zas what do you think?